### PR TITLE
Adding help script to copy journal data from one examiner to another …

### PR DESCRIPTION
**Pull request by andrew-lai-bjss**
…and one to output the formatted journal data

### CODE REVIEW

File: copy_examiner_journal.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    )
+except ClientError as e:
+    print(e.response['Error']['Message'])
+else:
+    item = response['Item']
+    journalData = item["journal"]
+    print("GetItem succeeded")
+
+# Update the entry in the to table
+response = to_table.update_item(
```
Comment (johnmorleybjss) [inline copy_examiner_journal.py]: Worth a comment to explain that we don't update the embedded staffNumber.

File: copy_examiner_journal.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    Key={
+        'staffNumber': to_staff
+    },
+    UpdateExpression="set journal=:j",
```
Comment (johnmorleybjss) [inline copy_examiner_journal.py]: Do `lastUpdatedAt` and `hash` come from the existing record? Could we not do a put operation to create a new item in the to_table whether one already exists or not?

File: copy_examiner_journal.py (hunk 1 above)
Comment (andrew-lai-bjss) [reply copy_examiner_journal.py]: Added comment to file

File: copy_examiner_journal.py (hunk 2 above)
Comment (andrew-lai-bjss) [reply copy_examiner_journal.py]: Possibly an improvement for the future.

**Pull request merged**